governance/handler.go

```diff
@@ -127,20 +173,43 @@ func (g *Governance) AddVote(key string, val interface{}) bool {
 
 func (g *Governance) adjustValueType(key string, val interface{}) interface{} {
 	k := GovernanceKeyMap[key]
-	reqType := GovernanceItems[k].t
 	var x interface{}
 
 	// When an int value comes from JS console, it comes as a float64
-	if reqType == uint64T && reflect.TypeOf(val) == float64T {
+	if GovernanceItems[k].checkValueType(uint64(0)) && reflect.TypeOf(val) == float64T {
 		x = uint64(val.(float64))
 		if float64(x.(uint64)) == val.(float64) {
 			return x
 		}
 		return val
 	}
 
+	if (k == params.AddValidator || k == params.RemoveValidator) && reflect.TypeOf(val) == stringT {
+		addresses := strings.Split(val.(string), ",")
+
+		if len(addresses) == 0 {
+			return val
+		}
+
+		if len(addresses) == 1 {
+			x = common.HexToAddress(val.(string))
+			return x
+		}
+
+		var nodeAddresses []common.Address
+		for _, str := range strings.Split(val.(string), ",") {
```

Comment:
```suggestion
		for _, str := range addresses {
```

I think we can use pre-defined local variable `addresses` in here.

Reply:
Thanks!! I reflected at https://github.com/klaytn/klaytn/pull/1110/commits/680be2ebb4e2fc84f7ecab4319657906761f9377.